tools/c7n_openstack/c7n_openstack/resources/resource_map.py

```diff
@@ -8,5 +8,6 @@
     "openstack.volume": "c7n_openstack.resources.volume.Volume",
     "openstack.security-group": "c7n_openstack.resources.security_group.SecurityGroup",
     "openstack.image": "c7n_openstack.resources.server.Image",
-    "openstack.secret": "c7n_openstack.resources.secret.Secret"
+    "openstack.secret": "c7n_openstack.resources.secret.Secret",
+    "openstack.container": "c7n_openstack.resources.object_storage.Container"
```

Comment:
I mean overall this looks good, just wondering about the name, container is pretty ambiguous. maybe storage-container..or just calling it a bucket 🤷‍♂️